sematic/types/types/dict.py

```diff
@@ -111,10 +112,15 @@ def _dict_to_json_encodable_summary(value: Dict, type_: Type) -> List[Tuple[Any,
     # implemented, since the keys must be hashable in order to be used as keys
     sorted_keys = sorted(value.keys(), key=hash)
 
-    return [
-        (
-            get_json_encodable_summary(key, key_type),
-            get_json_encodable_summary(value[key], element_type),
+    summary, blobs = [], {}
+
+    for key in sorted_keys:
+        key_summary, _ = get_json_encodable_summary(key, key_type)
```

Comment:
Are we guaranteed that keys will never have blobs? Admittedly it should be *rare*, but one could imagine something like a dict from images to a list of labels for it, as an example.

Reply:
I solemnly commit to addressing this issue personally when it happens 😬